dev/documentation/en/datamodel-settings.md

```diff
@@ -0,0 +1,80 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Settings - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: "Settings"
+
+Settings are additional property values associated with an instance of an entity type.
+
+## The Purpose of Settings
+
+Settings are an important concept in the PKP data model.
+They support two of the [overall goals](datamodel):
+
+### Localization
```

Comment:
I recommend incorporating the descriptive information here into the DataObject section of the [Entities](https://docs.pkp.sfu.ca/dev/documentation/en/architecture-entities#dataobject-class), maybe by splitting the `localizedData` examples into a different section with a brief description at the top.

A link should be added to the database page that explains the settings tables.